configuring.md

```diff
@@ -131,3 +131,13 @@ android {
   }
 }
 ```
+
+# ConscryptMode 
+
+Robolectric can currently either use Conscrypt and BouncyCastle or just BouncyCastle as the security provider. 
```

Comment:
Can this be `Starting in Robolectric 4.9, Robolectric can currently either use Conscrypt and BouncyCastle or just BouncyCastle as the security provider.`